contents/tutorials/product-data-in-new-tab.md

```diff
@@ -0,0 +1,79 @@
+
+### 1. A personal (or project) key
+
+We recommend creating a personal API key in PostHog for the API request. To do so, click your account icon in the top right corner, and then the gear next to your account name. In “My settings,” scroll down to “Personal API Keys” and click “Create personal API key.” Name it something like “Momentum.” Once created, make sure you copy and save it because you must create a new one if you lose it.
+
+![Personal API key](../images/tutorials/product-data-in-new-tab/personal-api-key.png)
+
+You can also use the project key (not recommended). You find it in “Project settings” under “Project Variables.”
+
+### 2. A project ID
+
+You find your project key by clicking “Project settings” in the sidebar, then scrolling down to the “Project Variables” and copying the value for “Project ID.” It should be a series of digits. 
+
+### 3. An insight short ID
+
+Next, you need the data you want to show in the new tab. To do so browse your insights or dashboards for a metric you want to display. When you find it, click on the insight details (the link from insights or “view” from a dashboard.” Once on that screen, copy the unique value in the URL after `<ph_instance_address>/insights/`.
+
+![Insight short ID](../images/tutorials/product-data-in-new-tab/short-id.png)
+
+## Formatting the API request
+
+Once you have all three pieces of information, we are going to use them to format an API request for the data Momentum Dash uses to display it. The API format is:
+
+```bash
+https://<ph_instance_address>/api/projects/<project_id>/insights/?personal_api_key=<personal_key>&short_id=<insight_short_id>
+```
+
+So if our instance address was `app.posthog.com`, project ID was `12345`, our personal API key was `phx_abcde`, and our insight short ID was `aAbBcC`, our API request would look like:
+
+```bash
+https://app.posthog.com/api/projects/12345/insights/?personal_api_key=phx_abcde&short_id=aAbBcC
+```
+
+Copy your formatted code and open up a new tab (assuming you have Momentum Dash set up) to move on to the next step.
+
+## Adding our metric in Momentum Dash
+
+Once you’ve signed up for Momentum Dash Plus and added the extension, your new tab page should look like this:
+
+![Momentum new tab](../images/tutorials/product-data-in-new-tab/new-tab.png)
+
+Hover your mouse in the top right corner, and an “add” button appears. Click it and select “Metric.” Next, choose “From a URL” and paste the API request you created in the last section into the URL field. If you formatted the API request properly, a drop-down appears that prompts you to “select an option.” 
+
+![Momentum API request](../images/tutorials/product-data-in-new-tab/momentum-api.png)
+
+When you click the drop-down, you get lots of options, you have to find the one that matches your metric (number) in PostHog. For my number chart, it was `result[0].result[0].aggregated_value` but it might also be `result[0].result[0].count` or another value. 
```

Comment:
```suggestion
When you click the drop-down, you get lots of options, you have to find the one that matches your metric (number) in PostHog. For my number chart, it was `result[0].result[0].aggregated_value` but it might also be `result[0].result[0].count`, or another value. 
```